packages/entity-browser/src/components/DetailView/LabeledField.js

```diff
@@ -53,7 +54,8 @@ LabeledField.propTypes = {
   meta: React.PropTypes.shape({
     touched: React.PropTypes.bool,
     error: React.PropTypes.objectOf(React.PropTypes.string)
-  })
+  }),
+  options: React.PropTypes.object
```

Comment:
can we define the `options` prop type a bit more precisely (i.e. by using shape()).